openedx/core/djangoapps/user_api/accounts/api.py

```diff
@@ -169,6 +173,12 @@ def update_account_settings(requesting_user, update, username=None):
                 "user_message": err.message
             }
 
+    if changing_full_name and check_html(update['name']):
+        field_errors["name"] = {
+            "developer_message": u"Error thrown from validate_full_name: '{}'".format('Full Name is in-valid'),
+            "user_message": u"This value is in-valid"
```

Comment:
Translate the user message.

Reply:
We need to communicate WHY it is invalid. (and as Waheed mentioned, translate it)

Prefer `Name cannot contain the following characters: < >`